client/src/components/Booking.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Stars from './Stars.jsx';
+import Form from './Form.jsx';
+import Finding from './Finding.jsx';
+import { Container } from 'semantic-ui-react';
+import styles from '../styles.css';
+
+
+const axios = require('axios');
+
+
+class Booking extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      room: {
+        room_id: this.props.room,
```

Comment:
You have a bunch of state variables that are never used, although I'm guessing you're gonna use these later on. Might want to remove the ones that you aren't using, though.

Reply:
@leealbert95 I removed all the variables except for room_id here. I need room_id variable to render it immediately.